apps/myaccount/src/helpers/user.ts

```diff
@@ -106,3 +106,15 @@ export const resolveUserStoreEmbeddedUsername = (username: string): string => {
 
     return username;
 };
+
+/**
+ * Resolves the user's userstore from the username
+ *
+ * @param {string} username - Username of the user with user store embedded.
+ * @return {string}
+ */
+export const resolveUserstore= (username: string): string => {
+    const parts = username.split("/");
+
+    return parts[0];
```

Comment:
Shall add a name for the index?

```suggestion
    // Userstore is index 0 and index 1 is username
    const USERSTORE = 0;
    const parts = username?.split("/");
    return parts[USERSTORE];
```

Reply:
Done